android-components/components/browser/state/src/main/java/mozilla/components/browser/state/engine/middleware/TranslationsMiddleware.kt

```diff
@@ -63,6 +64,11 @@ class TranslationsMiddleware(
                             requestSupportedLanguages(context, action.tabId)
                         }
                     }
+                    TranslationOperation.FETCH_DOWNLOADED_LANGUAGES -> {
+                        scope.launch {
+                            requestDownloadedLanguages(context)
```

Comment:
Please also include the `tabId = action.tabId` on this one.

Reply:
oh yeah, added `action.tabId`